docs/source/generate.rst

```diff
@@ -101,7 +109,13 @@ Browser
             loadFile("https://docxtemplater.com/tag-example.docx",function(error,content){
                 if (error) { throw error };
                 var zip = new PizZip(content);
-                var doc=new window.docxtemplater().loadZip(zip)
+                var doc;
+                try {
+                    doc=new window.docxtemplater(zip);
+                } catch(error) {
```

Comment:
Can you add a comment here : // Catch compilation errors (errors relating only with the template)